ui/src/app/shared/components/error-notice.tsx

```diff
@@ -1,13 +1,14 @@
 import * as React from 'react';
+import {CSSProperties} from 'react';
 import {Notice} from './notice';
+import {PhaseIcon} from './phase-icon';
 
 // Display an error notice.
 // If the error was a HTTP error (i.e. from super-agent), rather than just an unhelpful "Internal Server Error",
 // it will display any message in the body.
-export const ErrorNotice = (props: {error: Error & {response?: {body: {message?: string}}}}) => (
-    <Notice>
-        <i className='fa fa-exclamation-triangle status-icon--failed' />
-        {props.error.message}
+export const ErrorNotice = (props: {style?: CSSProperties; error: Error & {response?: {body: {message?: string}}}}) => (
+    <Notice style={props.style}>
+        <PhaseIcon value='Error' /> {props.error.message || 'Unknown error. Open your browser error console for more information.'}
```

Comment:
there are rare cases where we don't get info in the error - ask the user to look in the console